### PR TITLE
@craigspaeth: Marketing modals

### PR DESCRIPTION
**Pull request by xtina-starr**
closes: https://github.com/artsy/collector-experience/issues/314

This work updates the marketing modal to add a dedicated modal for the Art Basel 2017 campaign. This modal gets triggered on desktop and mobile web when a user navigates to the Art Basel 2017 Microsite or enters the `m-id=ca3` query param in the url.

*note:* updating the environment variables are pending.

**Desktop version:**
screen shot 2017-06-07 at 5 57 29 pm

**Mobile version:**
screen shot 2017-06-07 at 5 58 19 pm

### CODE REVIEW

File: desktop/components/marketing_signup_modal/index.coffee [hunk 1]
```diff
@@ -51,6 +51,11 @@ module.exports = class MarketingSignupModal extends Backbone.View
 
   initialize: ->
     slug = qs.parse(location.search.replace /^\?/, '')?['m-id']
+
+    # This is just for the Art Basel Fair campaign and should be removed after
+    # the fair closes.
+    slug = 'ca3' if !slug and sd.CURRENT_PATH is '/art-basel-2017'
```
Comment (xtina-starr) [inline desktop/components/marketing_signup_modal/index.coffee]: This is how I'm targeting the fair microsite landing page.

**Comment by xtina-starr** [comment]
I think I used the wrong image so I'm going to update that really quickly.

**Comment by xtina-starr** [comment]
There was notification spec that was failing so I'm trying to rebuild now.

**Comment by craigspaeth** [comment]
LGTM! Pretty straightforward. Looking forward to the PR for the modal management admin app 😉

**Comment by xtina-starr** [comment]
Haha me too!!

**Pull request merged**